### ISSUE TITLE
 if(html && html.match(/<h1>Sorry!<\/h1>/))

### ISSUE THREAD

**Issue by tambu22** [DoctorMcKay/node-steamcommunity#94]
i get this error when i try to use: 

```javascript
community.uploadAvatar('http://lorempixel.com/182/184/', function(err){ 
           console.log(err);
 }); 
```

after conecting with: 

```javascript
 var code = SteamTotp.generateAuthCode('XXXXXXXXXXXXXXXXXXXXXXXXXXX');
    community.login({
        "accountName": "User",
        "password": "Pw",
        "steamguard": "XXXX", // only required if logging in with a Steam Guard authorization
        "authCode": "XXXX", // only required if logging in with a new email auth code
        "twoFactorCode": code, // only required if logging in with a Steam Guard app code
     }, function(err,sessionID,cookies,steamguard ){
        	if(err) {
    			if(err.message == 'SteamGuardMobile') {
    				 logger.log("This account already has two-factor authentication enabled. " + err );
    				process.exit();
    				return;
    			}
    
    			if(err.message == 'SteamGuard') {
    				 logger.log("An email has been sent to your address at " + err.emaildomain);
    				rl.question("Steam Guard Code: ", function(code) {
    					doLogin(accountName, password, code);
    				});
    
    				return;
    			}
    
    			logger.log(err);    			
    			return;
    		}
    		 logger.log("Logged on!");
    		 Log2 =1;
    	    });
```

DIR/node_modules/steamcommunity/components/http.js:108
        if(html && html.match(/<h1>Sorry!<\/h1>/)) {
                        ^
 
TypeError: html.match is not a function
    at SteamCommunity._checkCommunityError (DIR/node_modules/steamcommunity/components/http.js:108:18)
    at Request._callback (DIR/node_modules/steamcommunity/components/http.js:51:88)
    at Request.self.callback (DIR/node_modules/request/request.js:199:22)
    at emitTwo (events.js:100:13)
    at Request.emit (events.js:185:7)
    at Request.<anonymous> (DIR/node_modules/request/request.js:1036:10)
    at emitOne (events.js:95:20)
    at Request.emit (events.js:182:7)
    at Gunzip.<anonymous> (DIR/node_modules/request/request.js:963:12)
    at emitNone (events.js:85:20)

**Comment by DoctorMcKay** [comment]
Fixed in v3.19.2. Sorry about that.

**Comment by tambu22** [comment]
now its work!! ty bro :+1:
